extensions/interactions/base_test.py

```diff
@@ -479,9 +480,16 @@ class InteractionDemoExplorationUnitTests(test_utils.GenericTestBase):
     _DEMO_EXPLORATION_ID = '16'
 
     def test_interactions_demo_exploration(self):
-        exp_services.load_demo(self._DEMO_EXPLORATION_ID)
-        exploration = exp_services.get_exploration_by_id(
-            self._DEMO_EXPLORATION_ID)
+        def mock_get_filename_with_dimensions(filename, unused_exp_id):
+            return exp_services.regenerate_image_filename_using_dimensions(
+                filename, 490, 120)
+
+        with self.swap(
```

Comment:
Again you should probably just upgrade the demo file.

Maybe that issue to file (mentioned above) can just upgrade _all_ the demo files to the latest schema version.

Reply:
Yeah will create one, and write a note there to remove these swap conditions from the tests too, wherever they are put before loading a demo exploration

Reply:
https://github.com/oppia/oppia/issues/5501